.github/workflows/workflow-reporter.yml

```
@@ -0,0 +1,24 @@
+workflowStat: true
+jobStat: true
+salutation: Hello
+body: The continuous integration workflow has failed. Please check the below details and resolve the issues.
+
+workflows:
```

actionlint (1.7.7)

unexpected key "workflows" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)